src/redux/topic/saga.js

```diff
@@ -92,9 +92,11 @@ export function* allowanceRequestHandler() {
 
       const result = yield call(request, `${bodhiapi}/allowance`, options);
 
+      const value = convertBNHexStrToQtum(result.remaining);
```

Comment:
nice change to have all the hex converted before being used by the UI 👍

Reply:
🙆‍♂️